app/views/admin/collections/_form.html.erb

```diff
@@ -46,15 +46,15 @@ Unless required by applicable law or agreed to in writing, software distributed
 <% content_for :page_scripts do %>
 <script>
   function inject_alert(message) {
-    $('#header-navbar').closest('.container-fluid').after(
+    $('.page-title').closest('.page-title-wrapper').before(
       '<div class="container-fluid" id="alerts"><div class="alert alert-danger"><button type="button" class="close" data-dismiss="alert">&times;</button>' +
       message + '</div></div>');
   }
 
   $('form[data-remote][modal]').on('ajax:complete', function (jq_event, xhr) {
     if (xhr.responseJSON.hasOwnProperty('errors')) {
       $(this).closest('.modal').modal('hide')
-      inject_alert(xhr.responseJSON['errors'].join('<br/>'));
+      inject_alert(xhr.responseJSON['errors'].join(' '));
```

Comment:
Does a space end up rendering better than a line break when there are multiple errors in the response?

Reply:
Oh it didn't cross my mind. But for the collections at the moment name is the only attribute that user can miss when submitting this form right?

Reply:
I think there can be multiple errors.  Most likely scenario where this might happen is if the collection name is missing or not unique and the contact email/url is improperly formatted.  I don't think it is possible to get an error on the unit or managers with the current UI.

https://github.com/avalonmediasystem/avalon/blob/develop/app/models/admin/collection.rb#L28-L32

Reply:
Yes :+1: I will fix this.